R/makeNeighbourhoods.R

```diff
@@ -30,63 +32,104 @@
 #' neighbourhoods.
 #'
 #' @author
-#' Mike Morgan, refined sampling written by Emma Dann
+#' Emma Dann, Mike Morgan
 #'
 #' @examples
 #'
 #' requires(igraph)
 #' m <- matrix(rnorm(10000), ncol=10)
 #' milo <- buildGraph(m, d=10)
 #'
-#' milo <- makeNeighbourhoods(milo, prop=0.3)
+#' milo <- makeNeighbourhoods(milo, prop=0.1)
 #' milo
 #' @name makeNeighbourhoods
-NULL
-
-#' @export
-#' @rdname makeNeighbourhoods
-#' @importFrom BiocNeighbors findKNN
-makeNeighbourhoods <- function(x, prop=0.1, k=21, refined=TRUE, seed=42, reduced_dims=NULL){
-
+makeNeighbourhoods <- function(x, prop=0.1, k=21, d=30, refined=TRUE, seed=42, reduced_dims="PCA") {
     if(class(x) == "Milo"){
         message("Checking valid object")
         # check that a graph has been built
-        if(!.valid_graph(graph(x))){
+        if(!.valid_graph(miloR::graph(x))){ ## Had to specify here for conflict w igraph
```

Comment:
igraph isn't imported in this function so I don't think it will matter.

Reply:
Will it matter if I have to import another function from igraph (igraph::neighbors in this case)?

Reply:
if you explicitly import the functions it shouldn't be an issue, e.g. 
```
#' @importFrom igraph neighbors
```